### PR TITLE
commit policy

### PR DESCRIPTION
**Pull request by sopos**
a description of the commit and review process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
 See:
 * https://github.com/psss/tmt
 * https://github.com/psss/fmf
+
+## Commit / merge policy
+
+Every important change to tests should undergo a review withing a pull request.
+
+### changes without a need of review
+Small, trivialy changes may be committed directly the master branch.
```
Comment (sergio-correia) [inline README.md:27]: typo: trivial

File: README.md [hunk 2]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
...
+All changes to the test logic (changes of expected output, expected exit codes) need to be reviewd.
+
+#### new tests
+It is recommended to push new tests via PR with review as well as a check for covering all the agreed acceptance criteria is more than welcome.
+
+#### reviewer selection
+The review can be done by any member of the team. However, it is recommended to ask the devel counterpart as they could judge the test expectations according to the actualy code change. Asking another QE person, on the other hand, will ansure to keep the test coding style as consistent as possible.
+
+#### PR merge
+The marge it self should be done based on the accepted reviews. It is recommended to let the original requestor to merge the PR as they may want to do some refinements, e.g. quash some commits which were added during the review process.
```
Comment (sergio-correia) [inline README.md:42]: typos: The merge itself / e.g. squash

File: README.md [hunk 3]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
 See:
 * https://github.com/psss/tmt
 * https://github.com/psss/fmf
+
+## Commit / merge policy
+
+Every important change to tests should undergo a review withing a pull request.
```
Comment (sopos) [inline README.md:24]:
```suggestion
Every important change to tests should undergo a review within a pull request.
```

File: README.md [hunk 4]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
 See:
 * https://github.com/psss/tmt
 * https://github.com/psss/fmf
+
+## Commit / merge policy
+
+Every important change to tests should undergo a review within a pull request.
+
+### changes without a need of review
+Small, trivialy changes may be committed directly the master branch.
```
Comment (sopos) [inline README.md:27]:
```suggestion
Small, trivial changes may be committed directly the master branch.
```

File: README.md [hunk 5]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
...
+As a typical trivial change one may considere a metadata update (updated tags, adjust, ..)
```
Comment (sopos) [inline README.md:28]:
```suggestion
As a typical trivial change one may consider a metadata update (updated tags, adjust, ..)
```

File: README.md [hunk 6]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
...
+or typo fixes, updated to comments and formating changes.
+
+### changes with a need of review
+#### existing tests
+All changes to the test logic (changes of expected output, expected exit codes) need to be reviewd.
```
Comment (sopos) [inline README.md:33]:
```suggestion
All changes to the test logic (changes of expected output, expected exit codes) need to be reviewed.
```

File: README.md [hunk 7]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
...
+
+#### new tests
+It is recommended to push new tests via PR with review as well as a check for covering all the agreed acceptance criteria is more than welcome.
+
+#### reviewer selection
+The review can be done by any member of the team. However, it is recommended to ask the devel counterpart as they could judge the test expectations according to the actualy code change. Asking another QE person, on the other hand, will ansure to keep the test coding style as consistent as possible.
```
Comment (sopos) [inline README.md:39]:
```suggestion
The review can be done by any member of the team. However, it is recommended to ask the devel counterpart as they could judge the test expectations according to the actual code change. Asking another QE person, on the other hand, will ensure to keep the test coding style as consistent as possible.
```

File: README.md [hunk 8]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
...
+
+#### PR merge
+The marge it self should be done based on the accepted reviews. It is recommended to let the original requestor to merge the PR as they may want to do some refinements, e.g. quash some commits which were added during the review process.
```
Comment (sopos) [inline README.md:42]:
```suggestion
The merge it self should be done based on the accepted reviews. It is recommended to let the original requestor to merge the PR as they may want to do some refinements, e.g. squash some commits which were added during the review process.
```

**Review by radosroka — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -18,3 +18,27 @@ The data are structured in flexible metadata format (fmf).
 See:
 * https://github.com/psss/tmt
 * https://github.com/psss/fmf
+
+## Commit / merge policy
+
+Every important change to tests should undergo a review within a pull request.
+
+### changes without a need of review
+Small, trivial changes may be committed directly the master branch.
```
Comment (ZelenyMartin) [inline README.md:27]: directly **to** the master branch

**Review by ZelenyMartin — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (sopos) [inline README.md:27]:
```suggestion
Small, trivial changes may be committed directly to the master branch.
```

**Review by ZelenyMartin — APPROVED** [review]

**Review by Cropi — APPROVED** [review]

**Pull request merged**